vultr/README.md

```diff
@@ -0,0 +1,32 @@
+# Vultr Command Set
+
+Vultr Server Command set
+
+## Available Commands
+
+- `vultr_list`- Lists Vultr server instances.
+- `vultr_reboot`- Reboot Vultr server instance.
+- `vultr_snap`- Create a snapshot of Vultr server instance.
+
+## Requirements
+
+To run this command set, we need a secret named `vultrApiKey`. This can be obtained by following ["This link"](https://docs.ansible.com/ansible/latest/scenario_guides/guide_vultr.html#authentication).
```

Comment:
```suggestion
To run this command set, we need a secret named `vultrApiKey`. This can be obtained by following this [guide](https://docs.ansible.com/ansible/latest/scenario_guides/guide_vultr.html#authentication).
```